castervoice/rules/apps/editor/unity.py

```diff
@@ -0,0 +1,64 @@
+from dragonfly import Key,MappingRule, Dictation, Choice,Text
+from castervoice.lib.merge.state.short import R
+from castervoice.lib.merge.additions import IntegerRef
+from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
+
+class UnityRule(MappingRule):
+	mapping = {
+		"<window>" : R(Key("c-%(window)s")),
```

Comment:
Personally, I prefer keywords before commands like this to reduce vocabulary clutter/conflict. The first thing that comes to mind is "go <window>". Is this supposed to open up new windows as well?

Reply:
Excellent point. Other suggestions could be "show" or "focus". I will think about what feels best.

(Triggering the shortcut for a window in Unity will shift focus to it or open it if it's not already part of the active layout.)